oozie/src/main/java/org/apache/falcon/oozie/feed/HCatReplicationWorkflowBuilder.java

```diff
@@ -72,6 +84,16 @@ public HCatReplicationWorkflowBuilder(Feed entity) {
             OozieUtils.unMarshalHiveAction(export);
         org.apache.falcon.oozie.hive.ACTION hiveExportAction = exportActionJaxbElement.getValue();
         addHDFSServersConfig(hiveExportAction, src, target);
+        Properties hiveConf = HiveUtil.getHiveCredentials(src);
```

Comment:
The action types are different for preprocess and hive actions with different classes - You can't pass a single base type to them.   But for Hive actions we can do a single function, but even there since we print action type specific logs, it will be a little bit of an overkill.   We can clean it up later